src/db.rs

```diff
@@ -38,315 +40,410 @@ impl Default for Connection {
     }
 }
 
-pub fn get_dbconn() -> Result<PooledConn> {
+pub async fn get_dbpool() -> Result<AnyPool> {
+    let db_type = env::var("KBS_DB_TYPE").expect("KBS_DB_TYPE not set");
     let host_name = env::var("KBS_DB_HOST").expect("KBS_DB_HOST not set");
     let user_name = env::var("KBS_DB_USER").expect("KBS_DB_USER not set.");
     let db_pw = env::var("KBS_DB_PW").expect("KBS_DB_PW not set.");
     let data_base = env::var("KBS_DB").expect("KBS_DB not set");
```

Comment:
for future PR: if KBS_DB_TYPE is sqlite, then don't fail when KBS_DB_HOST/KBS_DB_USER/KBS_DB_PW are not set (they are not used anyway).

Reply:
Yes, agreed.  I will fix this in a future pull request.  For sqlite, we shouldn't throw an error if host_name, user_name, and db_pw are not set.